src/tests/ftest/util/run_utils.py

```diff
@@ -154,7 +154,8 @@ def get_switch_user(user="root"):
     """
     command = ["sudo", "-n"]
     if user != "root":
-        command.extend(["-u", user])
+        # User runuser to avoid using a password
+        command.extend(["runuser", "-u", user])
```

Comment:
I think it's more reliable to add `--` to distinguish runuser arguments from the actual command
```suggestion
        command.extend(["runuser", "-u", user, "--"])
```

Reply:
Done.